Makefile

```diff
@@ -57,6 +57,7 @@ setup_env:
 	poetry run python -m pip install -U --pre --extra-index-url https://pypi.zama.ai/cpu "$(CONCRETE_PYTHON_VERSION)"
 	"$(MAKE)" fix_omp_issues_for_intel_mac
 	poetry run python -c "import skorch" || true # Details above
+	poetry run python -c "import brevitas" || true # Details above
```

Comment:
could you update https://github.com/zama-ai/concrete-ml-internal/issues/4272 by including the mention of brevitas + add a fixme here ? just realized we didn't put it last time